platform/vs/tests/breakout/test_breakout_cli.py

```diff
@@ -0,0 +1,148 @@
+from swsscommon import swsscommon
+import time
+import os
+import json
+import ast
+import pytest
+import collections
+
+@pytest.mark.usefixtures('dpb_setup_fixture')
+class TestBreakoutCli(object):
+    def setup_db(self, dvs):
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    def read_Json(self, dvs):
+        test_dir = os.path.dirname(os.path.realpath(__file__))
+        sample_output_file = os.path.join(test_dir, 'sample_output', 'sample_new_port_config.json')
+        fh = open(sample_output_file, 'rb')
+        fh_data = json.load(fh)
+        fh.close()
+
+        if not fh_data:
+            return False
+        expected = ast.literal_eval(json.dumps(fh_data))
+        return expected
+
+    def breakout(self, dvs, interface, brkout_mode):
+        (exitcode, result) = dvs.runcmd("config interface breakout {} {} -y".format(interface, brkout_mode))
+
+        if result.strip("\n")[0] == "[ERROR] Breakout feature is not available without platform.json file" :
+            pytest.exit("**** This test is not needed ****")
```

Comment:
I would recommend using `pytest.skip` here, just in case we're running this test alongside other test cases as well.

Reply:
#done